src/config/index.js

```diff
@@ -5,7 +5,7 @@ import colors from './colors'
 
 let config = {
   colors,
-  storybookEnabled: false,
+  storybookEnabled: true,
```

Comment:
We don't need storybookEnabled as a config option on the web and it should be in `local.js` anyway. We use that for values we change locally in development that could impact other collaborators.